src/macsniff.cpp

```diff
@@ -67,13 +68,10 @@ bool mac_add(uint8_t *paddr, int8_t rssi, bool sniff_type) {
             rgb_set_color(COLOR_NONE);
         }
      
-        if (added) { // first time seen this WIFI or BLE MAC
-            ESP_LOGI(TAG, "%s RSSI %ddBi -> Hash %04X -> WiFi:%d  BLE:%d  %s", 
-                                typebuff, rssi, hashedmac, 
-                                (int) wifis.size(), (int) bles.size(), buff );
-        } else { // already seen WIFI or BLE MAC
-            ESP_LOGI(TAG, "%s RSSI %ddBi -> Hash %04X -> already seen", typebuff, rssi, hashedmac);
-        }
+        ESP_LOGI(TAG, "%s RSSI %ddBi -> MAC %s -> Hash %04X -> WiFi:%d  BLE:%d  %s", 
+                        typebuff, rssi, buff, hashedmac, 
+                        (int) wifis.size(), (int) bles.size(),
+                        added ? "already seen" : "new");
```

Comment:
looks like "already seen" and "new" are swapped here

Reply:
ah ah yes correct ;-)